recipes/r-smartsnp/meta.yaml

```
@@ -0,0 +1,102 @@
+{% set version = '1.1.0' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-smartsnp
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/smartsnp_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/smartsnp/smartsnp_{{ version }}.tar.gz
+  sha256: a2c39153e425985b03427b1d1eab8a239b98272922ca49592c45c965b487ffbf
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}make
+    - {{ posix }}sed               # [win]
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+    - {{ stdlib("c") }}
+  host:
+    - r-base
+    - r-rspectra
+    - r-rcpp
+    - r-rcpparmadillo
+    - r-rfast
+    - r-bootsvd
+    - r-data.table
+    - r-foreach
+    - r-vegan
+    - r-vroom
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+    - r-rspectra
+    - r-rcpp
+    - r-rcpparmadillo
+    - r-rfast
+    - r-bootsvd
+    - r-data.table
+    - r-foreach
+    - r-vegan
+    - r-vroom
+
+test:
+  commands:
+    - $R -e "library('smartsnp')"           # [not win]
+    - "\"%R%\" -e \"library('smartsnp')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=smartsnp
+  license: MIT
+  summary: Fast computation of multivariate analyses of small (10s to 100s markers) to big (1000s
+    to 100000s) genotype data. Runs Principal Component Analysis allowing for centering,
+    z-score standardization and scaling for genetic drift, projection of ancient samples
+    to modern genetic space and multivariate tests for differences in group location
+    (Permutation-Based Multivariate Analysis of Variance) and dispersion (Permutation-Based
+    Multivariate Analysis of Dispersion).
+  license_family: MIT
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/MIT'
+    - LICENSE
+
+extra:
+  recipe-maintainers:
+    - TCLamnidis
```

R team would also like to maintain:

```suggestion
    - conda-forge/r
    - TCLamnidis
```